app/javascript/react/components/presentational/ScarcCoverflow/MediaGrid.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+
+function GridList({collections, selected_class, slide_clicked, on_load_all_images}) {
+  return(
+    <ul>
+      {collections.map((slide, i) => {
+        return (
+          <li key={`collection.${i}`} className={ `${selected_class(i)}` } data-index={i}>
+            <div className="selected-collection-header">
+              <h1 className="collection-name">{slide.collection.name}</h1>
+              <p className="collection-description">{slide.collection.detail}</p>
```

Comment:
`slide.collection.name` and `slide.collection.detail` need HTML decoded, so we can add HTML to the collection data and have it displayed here.